src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -3971,6 +3981,123 @@ public void TestLoadWithComment()
 
         #endregion
 
+        #region #nonnull
+
+        [Fact]
+        [Trait("Feature", "Directives")]
+        public void NonNullCSharp7_3()
+        {
+            var text = @"#nonnull restore";
+            var node = Parse(text, options: TestOptions.Regular7_3);
+            TestRoundTripping(node, text, disallowErrors: false);
+            VerifyErrorSpecial(node, new DirectiveInfo { Number = (int)ErrorCode.ERR_FeatureNotAvailableInVersion7_3, Status = NodeStatus.IsError });
```

Comment:
>Status = NodeStatus.IsError [](start = 118, length = 27)

This feels unexpected. I think we still want to recognize the directive and take it into account even though the language version is not what we would expect.

Reply:
It looks like I misinterpreted what ```VerifyErrorSpecial``` is checking for

---
In reply to: [224871792](https://github.com/dotnet/roslyn/pull/30456#discussion_r224871792) [](ancestors = 224871792)